cmd/gateway/azure/gateway-azure.go

```diff
@@ -553,7 +658,9 @@ func (a *azureObjects) ListObjects(ctx context.Context, bucket, prefix, marker,
 
 	result.Objects = objects
 	result.Prefixes = prefixes
-	result.NextMarker = marker
+	if marker != "" {
+		result.NextMarker = azureMarkerPrefix + marker
```

Comment:
Unable to understand the use of prefixing `azureMarkerPrefix`.  Could you explain?

Reply:
This comment explains:
```
606         // To accommodate S3-compatible applications using                                                                                                                                                     
607         // ListObjectsV1 to use object keys as markers to control the                                                                                                                                          
608         // listing of objects, we use the following encoding scheme to                                                                                                                                         
609         // distinguish between Azure continuation tokens and application                                                                                                                                       
610         // supplied markers.                                                                                                                                                                                   
611         //                                                                                                                                                                                                     
612         // - NextMarker in ListObjectsV1 response is constructed by                                                                                                                                            
613         //   prefixing "{minio}" to the Azure continuation token,                                                                                                                                              
614         //   e.g, "{minio}CgRvYmoz"                                                                                                                                                                            
615         //                                                                                                                                                                                                     
616         // - Application supplied markers are used as-is to list                                                                                                                                               
617         //   object keys that appear after it in the lexicographical order.                                                                                                                                    
```